e2e/data.ts

```diff
@@ -54,48 +26,15 @@ const data = {
 			joinCode: '123'
 		}
 	},
-	userRegularChannels: {
-		detoxpublic: {
-			name: `detox-public-${value}`
-		}
-	},
-	groups: {
-		private: {
-			name: `detox-private-${value}`
-		},
-		alternate: {
-			name: `detox-alternate-${value}`
-		},
-		alternate2: {
-			name: `detox-alternate2-${value}`
-		}
-	},
-	teams: {
-		private: {
-			name: `detox-team-${value}`
-		}
-	},
-	registeringUser: {
-		username: `newuser${value}`,
-		password: `password${value}`,
-		email: `mobile+registering${value}@rocket.chat`
-	},
-	registeringUser2: {
-		username: `newusertwo${value}`,
-		password: `passwordtwo${value}`,
-		email: `mobile+registeringtwo${value}@rocket.chat`
-	},
-	registeringUser3: {
-		username: `newuserthree${value}`,
-		password: `passwordthree${value}`,
-		email: `mobile+registeringthree${value}@rocket.chat`
-	},
-	registeringUser4: {
-		username: `newuserfour${value}`,
-		password: `passwordfour${value}`,
-		email: `mobile+registeringfour${value}@rocket.chat`
-	},
-	random: value
+	randomUser: (): { username: string; name: string; password: string; email: string } => {
+		const randomVal = random();
+		return {
+			username: `user${randomVal}`,
+			name: `user${randomVal}`, // FIXME: apply a different name
```

Comment:
Will this stay for the next version?

Reply:
Yes, we need to fix a few components first.
I didn't want to do it on this PR, because it's already too big.